core/shared/platform/common/posix/posix_socket.c

```diff
@@ -906,7 +906,7 @@ os_socket_set_ip_multicast_ttl(bh_socket_t socket, uint8_t ttl_s)
 int
 os_socket_get_ip_multicast_ttl(bh_socket_t socket, uint8_t *ttl_s)
 {
-    socklen_t opt_len = sizeof(ttl_s);
+    socklen_t opt_len = sizeof(*ttl_s);
```

Comment:
sizeof(int)?

Reply:
Same as above, `sizeof(ttl_s)` is used in L898.